jni/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.6.0-dev-2
```

Comment:
I recently learned we should use `-dev` for when making an actual release, and `-wip` for landing commits that we don't make a release for yet.

https://github.com/dart-lang/native/pull/89#issuecomment-1633204927

I believe this should be `-wip` until we remove the `-wip` and release `0.6.0`. We're not intending to release a `0.6.0-dev-2` before `0.6.0`.